### DISCUSSION TITLE
npm lists 1 package in my profile but i cant see or edit / delete it

### DISCUSSION THREAD

**Discussion by CypherpunkSamurai** [community/community#146479] [npm]
I have an npm account with the same username as my current username, and I have tried to publish 1 package yesterday then deleted it cause versioning was wrong. today when i retried to publish the package i cant publish it anymore and have 1 hidden package in my account, and i cant see or delete it

![image](https://github.com/user-attachments/assets/87b9f234-bc2c-4fd4-bd5c-d6865c5a8d5e)

**Comment by Pilaton** [comment]
Have you tried removing it [via CLI](https://docs.npmjs.com/policies/unpublish#how-to-unpublish)?
